delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/client/S3StorageClient.scala

```diff
@@ -57,6 +57,8 @@ trait S3StorageClient {
   def bucketExists(bucket: String): IO[Boolean]
 
   def baseEndpoint: Uri
+
+  def prefix: Uri
```

Comment:
I don't like this but it at least keeps the optionality of S3 storages to one place (otherwise you'd need to inject optionally to the call site)